misc/docerina/src/main/resources/template/html/module.hbs

```diff
@@ -17,10 +17,7 @@
 
     <div class="fourteen wide column content-wrapper">
         <div class="main-wrapper module-wrapper">
-            <div class="ui breadcrumb">
-                <h1 class="active section">Module : <span class="modules">ballerina/{{ module.id }}</span></h1>
-            </div>
-            <div class="ui divider"></div>
+                <h1>Module : <span class="modules">ballerina/{{ module.id }}</span></h1>
```

Comment:
We should avoid hardcoding ballerina/
It's available with {{ project.organization }}